packages/charting/src/demo/GettingStartedPage.tsx

```diff
@@ -0,0 +1,108 @@
+/* tslint:disable:jsx-no-multiline-js max-line-length */
+
+import * as React from 'react';
+import { Link } from 'office-ui-fabric-react/lib/Link';
+import './GettingStartedPage.scss';
+import { Highlight } from '@uifabric/example-app-base';
+
+export class GettingStartedPage extends React.Component<{}, {}> {
+  public render(): JSX.Element {
+    return (
+      <div className="ms-GettingStartedPage">
+        <div className="ms-GettingStartedPage-banner">
+          <h1>Office UI Fabric React Charting!</h1>
+          <h3>A library of reusable, generic React components</h3>
+        </div>
+
+        <h2>Overview</h2>
+
+        <p>
+          Fabric React components are built as production ready components to be used in Microsoft products, but
+          generalized, documented, and reusable. This enables us and our partners to more easily build great
+          applications without spending a ton of time implementing the same things over and over.`
+        </p>
+
+        <p>
+          Each component is designed to be RTL friendly, keyboard accessible, screen reader friendly, themeable, and
+          generalized. TypeScript definition files are also included, so if you use TypeScript (which isn't a
+          requirement), you will get compiler validation and using an editor like VS Code, you'll get intellisense. Each
+          component is exported as a named module that can be easily imported in your code, allowing your external
+          bundler to create small bundles that include just what you need.`
+        </p>
+
+        <h2>Getting started</h2>
+
+        <p>
+          Integrating components into your project depends heavily on your setup. The recommended setup is to use a
+          bundler such as
+          <Link href="https://webpack.github.io/" target="_blank">
+            Webpack
+          </Link>
+          which can resolve NPM package imports in your code and can bundle the specific things you import.
+        </p>
+
+        <p>Within an npm project, you should install the package and save it as a dependency:</p>
+
+        <div className="ms-GettingStartedPage-code">
+          <Highlight className="bash">npm install --save office-ui-fabric-react</Highlight>
```

Comment:
>office-ui-fabric-react [](start = 57, length = 22)

A lot of the things on this page is not in sync with the package/page title..

Reply:
This is the same as the Getting Started page from the experiments package, which also doesn't seem to match up with what's in experiments. I think updating these pages to match their packages is a separate work item.